pkg/virtual/workspaces/builder/build.go

```diff
@@ -47,7 +48,7 @@ import (
 	"github.com/kcp-dev/kcp/pkg/virtual/workspaces/registry"
 )
 
-func BuildVirtualWorkspace(rootPathPrefix string, wildcardsClusterWorkspaces workspaceinformer.ClusterWorkspaceInformer, wildcardsRbacInformers rbacinformers.Interface, kubeClusterClient kubernetes.ClusterInterface, kcpClusterClient kcpclient.ClusterInterface) framework.VirtualWorkspace {
+func BuildVirtualWorkspace(kubeClientConfig *clientrest.Config, rootPathPrefix string, wildcardsClusterWorkspaces workspaceinformer.ClusterWorkspaceInformer, wildcardsRbacInformers rbacinformers.Interface, kubeClusterClient kubernetes.ClusterInterface, kcpClusterClient kcpclient.ClusterInterface) framework.VirtualWorkspace {
```

Comment:
which rest config is that? A cluster config? Then please call it like that. Why is there also a kubeClusterClient?

Reply:
it's just the config passed to all VirtualWorkspaces.

Let me name it `cfg`, the same as @stevekuznetsov did in the `initializingworkspaces` VW:
```
func BuildVirtualWorkspace(
	cfg *rest.Config,
	rootPathPrefix string,
	dynamicClusterClient dynamic.ClusterInterface,
	kubeClusterClient kubernetes.ClusterInterface,
	wildcardKcpInformers kcpinformer.SharedInformerFactory,
) (map[string]framework.VirtualWorkspace, error) {
```

Reply:
it's simply named "config" in the upper calls.

Reply:
Fixed in commit https://github.com/kcp-dev/kcp/pull/1456/commits/001ae2f79dcd0fa6e0610661143936c0bc82898e or PR #1456